includes/course-theme/class-sensei-course-theme-quiz.php

```diff
@@ -58,23 +58,24 @@ public function init() {
 	 */
 	private function maybe_add_quiz_results_notice() {
 		$lesson_id = Sensei_Utils::get_current_lesson();
-		$quiz_id   = \Sensei()->lesson->lesson_quizzes( $lesson_id );
+		$quiz_id   = Sensei()->lesson->lesson_quizzes( $lesson_id );
 		$user_id   = get_current_user_id();
 
 		if ( empty( $user_id ) ) {
 			return;
 		}
 
-		$quiz_status = Sensei_Utils::user_lesson_status( $lesson_id, $user_id )->comment_approved;
+		$user_lesson_status = Sensei_Utils::user_lesson_status( $lesson_id, $user_id );
 
 		// If quiz is not submitted, then nothing else to do.
-		if ( ! \Sensei()->lesson->is_quiz_submitted( $lesson_id, $user_id ) ) {
+		if ( ! Sensei()->lesson->is_quiz_submitted( $lesson_id, $user_id ) || empty( $user_lesson_status ) ) {
```

Comment:
Basically, this PR just checks if `$user_lesson_status` exists before getting the `->comment_approved` attribute.
The rest are some PHPCS fixes.